src/components/Markdown/Markdown.style.ts

```diff
@@ -126,6 +127,16 @@ export const StylesMarkdown = css`
   blockquote {
     ${blockquoteCss};
   }
+
+  a {
```

Comment:
implementing in App.css didn't do much. i think editing markdown styles targets what we want to target (learn, about page, data api page)